spec/airport.spec.js

```diff
@@ -0,0 +1,174 @@
+import Airport from "../src/airport.js";
+import Plane from "../src/Plane.js";
+import Weather from "../src/Weather.js";
+
+describe('Airport Tests', () => {
```

Comment:
Could have used a separate describe test block for each story to make the code more organised and easier to read.